packages/perseus-core/src/widgets/matcher/matcher-util.test.ts

```diff
@@ -0,0 +1,54 @@
+import getMatcherPublicWidgetOptions from "./matcher-util";
+
+import type {PerseusMatcherWidgetOptions} from "../../data-schema";
+
+describe("getMatcherPublicWidgetOptions", () => {
+    it("should return shuffled right array when order doesn't matter", () => {
+        // Arrange
+        const options: PerseusMatcherWidgetOptions = {
+            labels: ["**Claims**", "**Evidence**"],
+            padding: true,
+            orderMatters: false,
+            right: ["Stars", "Earth", "Life", "Rapid", "Milky Way"],
+            left: ["Fuel", "Plate", "Hydrogen", "Average", "Billion"],
+        };
+
+        // Act
+        const publicWidgetOptions = getMatcherPublicWidgetOptions(options);
+
+        // Assert
+        expect(publicWidgetOptions).toEqual({
+            labels: ["**Claims**", "**Evidence**"],
+            padding: true,
+            orderMatters: false,
+            right: ["Milky Way", "Stars", "Life", "Earth", "Rapid"],
+            left: ["Fuel", "Plate", "Hydrogen", "Average", "Billion"],
+        });
+        expect(publicWidgetOptions.right).not.toEqual(options.right);
+    });
+
+    it("should return shuffled left and right arrays when order matters", () => {
```

Comment:
I read your Github comment further down this test. Given that, I wonder if we could be more accurate with this in saying `should not return the left or right arrays in answer order`?

Reply:
🤔  Yeah, I guess "negation" in test names is kinda like naming a variable like `notEnabled`.  Boolean logic with it is a nightmare `if (!notEnabled) { /* enabled! */ }`.

Reply:
I updated it to `should return a shuffled right array when order doesn't matter to remove the correct order information` which I think is more clear as it gets rid of the negation while still indicating the goal is to remove correct order info